benchmark/single-source/DataBenchmarks.swift

```diff
@@ -56,72 +58,90 @@ public let DataBenchmarks = [
-    replace($0*M, data: medium, subrange:431..<809, with: small) }, tags: d),
+    replace($0*M/100, data: medium, subrange:431..<809, with: small) },
+    tags: d, legacyFactor: 100),
   BenchmarkInfo(name: "DataReplaceMedium", runFunction: {
-    replace($0*M, data: medium, subrange:431..<809, with: medium) }, tags: d),
+    replace($0*M/100, data: medium, subrange:431..<809, with: medium) },
+    tags: d, legacyFactor: 100),
   BenchmarkInfo(name: "DataReplaceLarge", runFunction: {
-    replace($0*M, data: medium, subrange:431..<809, with: large) }, tags: d),
+    replace($0*M/100, data: medium, subrange:431..<809, with: large) },
+    tags: d, legacyFactor: 100),
 
   BenchmarkInfo(name: "DataReplaceSmallBuffer", runFunction: {
-    replaceBuffer($0*M, data: medium, subrange:431..<809, with: small) },
-    tags: d),
+    replaceBuffer($0*M/100, data: medium, subrange:431..<809, with: small) },
+    tags: d, legacyFactor: 100),
   BenchmarkInfo(name: "DataReplaceMediumBuffer", runFunction: {
-    replaceBuffer($0*M, data: medium, subrange:431..<809, with: medium) },
-    tags: d),
+    replaceBuffer($0*M/100, data: medium, subrange:431..<809, with: medium) },
+    tags: d, legacyFactor: 100),
   BenchmarkInfo(name: "DataReplaceLargeBuffer", runFunction: {
-    replaceBuffer($0*M, data: medium, subrange:431..<809, with: large) },
+    replaceBuffer($0*10, data: medium, subrange:431..<809, with: large) },
     tags: d),
 
   BenchmarkInfo(name: "DataAppendSequence",
-    runFunction: { append($0*M, sequenceLength: 809, to: medium) }, tags: d),
+    runFunction: { append($0*M/100, sequenceLength: 809, to: medium) },
+  tags: d, legacyFactor: 100),
 
   BenchmarkInfo(name: "DataAppendDataSmallToSmall",
-    runFunction: { append($0*M, data: small, to: small) }, tags: d),
+    runFunction: { append($0*M/20, data: small, to: small) }, tags: d,
+    legacyFactor: 20),
   BenchmarkInfo(name: "DataAppendDataSmallToMedium",
-    runFunction: { append($0*M, data: small, to: medium) }, tags: d),
+    runFunction: { append($0*M/20, data: small, to: medium) }, tags: d,
+    legacyFactor: 20),
   BenchmarkInfo(name: "DataAppendDataSmallToLarge",
-    runFunction: { append($0*M, data: small, to: large) }, tags: d),
+    runFunction: { append($0*M/200, data: small, to: large) }, tags: d,
+    legacyFactor: 200),
   BenchmarkInfo(name: "DataAppendDataMediumToSmall",
-    runFunction: { append($0*M, data: medium, to: small) }, tags: d),
+    runFunction: { append($0*M/20, data: medium, to: small) }, tags: d,
+    legacyFactor: 20),
   BenchmarkInfo(name: "DataAppendDataMediumToMedium",
-    runFunction: { append($0*M, data: medium, to: medium) }, tags: d),
+    runFunction: { append($0*M/20, data: medium, to: medium) }, tags: d,
+    legacyFactor: 20),
   BenchmarkInfo(name: "DataAppendDataMediumToLarge",
-    runFunction: { append($0*M, data: medium, to: large) }, tags: d),
+    runFunction: { append($0*50, data: medium, to: large) }, tags: d),
```

Comment:
Oh, I see that these are no longer dependent on `M` at all, and are thus not adjusted by the legacy factor?

Reply:
I didn't add the factor here, because it was `.skip`ped before my refactoring. But since it was historically in the "database" before it was accidentally(?) disabled with other `Large` tests, I should probably add the `legacyFactor` anyway.

Reply:
@eeckstein ^^ Thoughts?

Reply:
I'm adding LF for these two cases too. Can't hurt ;-)